flink-table/flink-table-runtime-blink/pom.xml

```diff
@@ -61,6 +61,13 @@ under the License.
 			<version>${janino.version}</version>
 		</dependency>
 
+		<!-- Lz4 compression library -->
```

Comment:
@KurtYoung please also update the LICENSE file. Here is a rough description: https://cwiki.apache.org/confluence/display/FLINK/Licensing

Reply:
ok, will do